src/com/theironyard/novauc/Car.java

```diff
@@ -0,0 +1,55 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by octavio on 2/7/17.
+ */
+public class Car {
+    String brand;
+    String model;
+    int horsepower;
+    boolean originalOwner;
+    boolean moreThan100K;
+
+    public String getBrand() {
+        return brand;
+    }
+
+    public void setBrand(String brand) {
+        this.brand = brand;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public int getHorsepower() {
+        return horsepower;
+    }
+
+    public void setHorsepower(int horsepower) {
+        this.horsepower = horsepower;
+        if (horsepower > 250) {
+            System.out.println("Wow, that's a lot of horsepower dude!");
+        }
```

Comment:
Octavio, where are the constructors for each class?  You also need to add constraints to the other 4 classes.  The constraint you wrote on lines 33-38 is well done.  As is the rest of the project.  Please add those missing requirements and resubmit your assignment